org-roam-db.el

```diff
@@ -186,6 +186,8 @@ The query is expected to be able to fail, in this situation, run HANDLER."
 (defconst org-roam-db--table-schemata
   '((files
      [(file :unique :primary-key)
+      ;; "file-title" would be converted to "file_title"
```

Comment:
remove this comment. Also I think since it's the `files` table, `title` is sufficient, rather than `filetitle`.